apps/frontend/package.json

```diff
@@ -48,6 +48,7 @@
     "eslint-config-next": "14.2.13",
     "postcss": "^8",
     "tailwindcss": "^3.4.1",
-    "typescript": "^5"
+    "typescript": "^5",
+    "zxcvbn": "^4.4.2"
```

Comment:
does it make sense to use typescript version of this instead? https://www.npmjs.com/package/@zxcvbn-ts/core

Reply:
Thanks for finding, looked and didn't find it originally.